pypuppetdb/types.py

```diff
@@ -329,8 +329,8 @@ class Node(object):
             collected.
     :type facts_timestamp: :obj:`string` formatted as\
             ``%Y-%m-%dT%H:%M:%S.%fZ``
-    :param status: (default `None`) Status of the node\
-            changed | unchanged | unreported | failed
+    :param status_report: (default `None`) Status of latest report \
```

Comment:
Hum. I'm fairly sure I left the tests green (https://github.com/voxpupuli/pypuppetdb/pull/109/commits) , so let me double check it!